river/command/map.zig

```diff
@@ -210,6 +261,58 @@ fn parseModifiers(modifiers_str: []const u8, out: *?[]const u8) !wlr.Keyboard.Mo
     return modifiers;
 }
 
+fn parseSwitchType(
+    switch_type_str: []const u8,
+    out: *?[]const u8,
+) !Switch.Type {
+    if (mem.eql(u8, switch_type_str, "lid")) {
+        return .lid;
+    } else if (mem.eql(u8, switch_type_str, "tablet")) {
+        return .tablet;
+    }
+    out.* = try std.fmt.allocPrint(
+        util.gpa,
+        "invalid switch '{s}', must be 'lid' or 'tablet'",
+        .{switch_type_str},
+    );
+    return Error.Other;
+}
```

Comment:
This function could be replaced with `std.meta.stringToEnum(Switch.Type, str) orelse { ... }`

Reply:
For now I just changed the contents of the function and not the function itself to avoid duplication in `mapSwitch` and `unmapSwitch`. However it's just a few lines so maybe you prefer them inline?